### PR TITLE
update docker image version

### LINKED ISSUE #214: nfs-common install issue
**What happened**:

https://github.com/kubernetes-sigs/blob-csi-driver/blob/06df5fee08512fc17cc3684173b3b075fee7dec9/pkg/blobplugin/Dockerfile#L28-L29

```
#16 9.211 Setting up nfs-common (1:1.3.4-2.5+deb10u1) ...
#16 9.233 /var/lib/dpkg/info/nfs-common.postinst: 7: /var/lib/dpkg/info/nfs-common.postinst: ucf: not found
#16 9.233 dpkg: error processing package nfs-common (--configure):
#16 9.233  installed nfs-common package post-installation script subprocess returned error exit status 127
#16 9.234 Processing triggers for libc-bin (2.28-10) ...
#16 9.266 Errors were encountered while processing:
#16 9.266  nfs-common
#16 9.302 E: Sub-process /usr/bin/dpkg returned an error code (1)
#16 ERROR: executor failed running [/bin/sh -c apt update && apt install mount nfs-common -y]: buildkit-runc did not terminate successfully
------
 > [stage-1 6/6] RUN apt update && apt install mount nfs-common -y:
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c apt update && apt install mount nfs-common -y]: buildkit-runc did not terminate successfully
```

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
- CSI Driver version:
- Kubernetes version (use `kubectl version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**
update docker image version would also update the kernel version. That could possibly fix #214 and #220 
and also in go.mod we are using the version as 1.16 but the image used for building is still 1.13. Hence updating. 

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
thanks for the contribution!

**Comment by andyzhangx** [comment]
@boddumanohar can you also update the azure file repo? thanks.

File: pkg/blobplugin/Dockerfile [hunk 1]
```diff
@@ -26,7 +26,8 @@ RUN apt-get update && clean-install ca-certificates pkg-config libfuse-dev cmake
 RUN wget -O /tmp/packages-microsoft-prod.deb https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb && dpkg -i /tmp/packages-microsoft-prod.deb && apt-get update && apt install blobfuse fuse -y && rm -f /tmp/packages-microsoft-prod.deb
 RUN apt remove wget -y
 # this is a workaround to install nfs-common & nfs-kernel-server and don't quit with error
-RUN apt update && apt install nfs-common nfs-kernel-server -y || true
+# WIP: check if updating the kernel version would fix the below error
+# RUN apt update && apt install nfs-common nfs-kernel-server -y || true
```
Comment (andyzhangx) [inline pkg/blobplugin/Dockerfile:30]: wait, it's not installing `nfs-common`, that's in correct. I will start a new PR to fix this issue.

Comment (boddumanohar) [reply pkg/blobplugin/Dockerfile:30]: but without installing `nfs-common` this the tests have passed. That means that its no longer needed right?

Comment (andyzhangx) [reply pkg/blobplugin/Dockerfile:30]: no, there is no nfs e2e tests now

Comment (boddumanohar) [reply pkg/blobplugin/Dockerfile:30]: oh! ok. let me create an other PR to revert this change.

Comment (boddumanohar) [reply pkg/blobplugin/Dockerfile:30]: hoo, I see you have already made that change :). Cool.

**Pull request merged**